### ISSUE TITLE
UI elements missing

### ISSUE THREAD

**Issue by insunaa** [MNoya/PMP#24]
Since the latest update the upgrade cost of abilities (stun, dodge, racial, etc.) as well as the upgrade cost of weapons and armor etc. are missing. So is the portrait of the demon race.

(I haven't been able to play for 2 weeks, so I was unable to test the game before that)
![interface](https://cloud.githubusercontent.com/assets/3718129/16539747/520baf9c-404c-11e6-99c6-22ac6304a05e.png)

I've restarted dota and checked file integrity, but that didn't help, so I assume it's an issue with pmp :)

**Comment by MNoya** [comment]
Yeah, I had to remove that UI overlay because Valve changed something and it started crashing the game (it was an old module in scaleform). I kinda need to search for an anternative, which I think it can't be anything else than a small image edit directly onto the png file (like on the goblin tech upgrade).

**Comment by insunaa** [comment]
Oh, alright.
